### PR TITLE
feat: Add spec and parser for systemctl_status_-all

### PR DESCRIPTION
**Pull request by shlao**
Signed-off-by: shlao <shlao@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

The Open Shift Node will fail to start the `cri-o` service with the error messages showing in the command `systemctl status --all`.  Detail info please see: INSIGHTOCP-525 card.

### CODE REVIEW

**Comment by shlao** [comment]
@xiangce Will you help me review the MR? Thanks!

File: insights/parsers/systemctl_status_all.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""
+SystemctlStatusAll file ``sos_commands/systemd/systemctl_status_--all``
```
Comment (xiangce) [inline insights/parsers/systemctl_status_all.py:2]: Better modify as:
`SystemctlStatusAll command ``systemctl status --all``

File: insights/parsers/systemctl_status_all.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+from insights.core import LogFileOutput
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.systemctl_status_all)
+class SystemctlStatusAll(LogFileOutput):
+    """
+    Class for parsing the log file:
+    ``/sos_commands/systemd/systemctl_status_--all``.
```
Comment (xiangce) [inline insights/parsers/systemctl_status_all.py:15]: the same comment as above.

**Review by xiangce — CHANGES_REQUESTED** [review]
Please add a short description about why add a sos parser to the insights core

**Review by xiangce — APPROVED** [review]

**Pull request merged**